lib/pages/portal/service/website_provider.dart

```diff
@@ -229,31 +240,36 @@ class WebsiteProvider with ChangeNotifier {
     }
   }
 
+  List<String> getUserSources() =>
+      _authProvider.currentUserFromCache?.sourcesList;
+
```

Comment:
Nit: this could probably be removed, since we already have filterBySource and we can fetch it there.
Not a strong opinion though, feel free to leave it as is.

Reply:
I think the current version makes code a little more clear, so I feel like not changing it.